charts/dave/values.yaml

```diff
@@ -54,7 +54,7 @@ backend:
       SPRING_DATASOURCE_URL: jdbc:postgresql://dave-postgresql:5432/dave_db
       SPRING_DATASOURCE_USERNAME: dave
       SPRING_JPA_DATABASE: postgresql
-      SPRING_JPA_HIBERNATE_DDL-AUTO: update
+      SPRING_JPA_HIBERNATE_DDL-AUTO: validate
```

Comment:
redundenat https://github.com/it-at-m/helm-charts/pull/95/files#diff-177fa935390658f0b17aceed9e4dde3138e50dc718e4986aaa25fc674cf94970L57